tests/requirement/test_requirement_endo_bleeding.py

```
@@ -21,6 +21,11 @@
     generate_patient
 )
 
+from ..test_settings import (
+    ASSET_DIR,
```

Ruff (0.8.2)

`..test_settings.ASSET_DIR` imported but unused

Remove unused import: `..test_settings.ASSET_DIR`

(F401)